File: templates/Uwp/_composition/MVVMLight/Page.TabbedPivot.AddBehavior/Helpers/PivotItemExtensions.cs
using System;
using Windows.UI.Xaml.Controls;

namespace Param_ItemNamespace.Helpers
{
    public static class PivotItemExtensions
    {
        public static T GetPage<T>(this PivotItem pivotItem)
        {
            if (pivotItem.Content is Frame frame)
            {
                if (frame.Content is T element)
                {
                    return element;
                }
            }

            return default(T);
        }

        public static bool IsOfViewModelName(this PivotItem pivotItem, string viewModelName)
        {
            if (pivotItem.Content is Frame frame)
            {
                if (frame.Content is Page page)
                {
                    if (page.DataContext.GetType().FullName == viewModelName)
                    {
                        return true;
                    }
                }
            }

            return false;
        }
    }
}

File: templates/Uwp/_composition/MVVMLight/Project.TabbedPivotVB/Helpers/PivotItemExtensions.vb
Namespace Helpers
    Friend Module PivotItemExtensions
        <Extension()>
        Function GetPage(Of T As Class)(pivotItem As PivotItem) As T
            Dim frame = TryCast(pivotItem.Content, Frame)
            If frame IsNot Nothing Then
                Dim element = TryCast(frame.Content, T)
                If element IsNot Nothing Then
                    Return element
                End If
            End If

            Return Nothing
        End Function

        <Extension()>
        Function IsOfViewModelName(pivotItem As PivotItem, viewModelName As String) As Boolean
            Dim frame = TryCast(pivotItem.Content, Frame)
            If frame IsNot Nothing Then
                Dim page = TryCast(frame.Content, Page)
                If page IsNot Nothing Then
                    If page.DataContext.GetType().FullName = viewModelName Then
                        Return True
                    End If
                End If
            End If

            Return False
        End Function
    End Module
End Namespace